addons/mail/static/tests/owl/component/chatter_topbar_tests.js

```diff
@@ -2,28 +2,28 @@ odoo.define('mail.component.ChatterTopBarTests', function (require) {
 'use strict';
 
 const ChatterTopBar = require('mail.component.ChatterTopbar');
+const { makeTestPromise } = require('web.test_utils');
 const {
     afterEach: utilsAfterEach,
+    afterNextRender,
     beforeEach: utilsBeforeEach,
     pause,
     start: utilsStart,
 } = require('mail.messagingTestUtils');
 
+
 QUnit.module('mail.messaging', {}, function () {
 QUnit.module('component', {}, function () {
 QUnit.module('ChatterTopbar', {
     beforeEach() {
         utilsBeforeEach(this);
-        this.createThread = async ({ _model, id }, { fetchAttachments=false }={}) => {
-            const threadLocalId = this.env.store.dispatch('_createThread', { _model, id });
-            if (fetchAttachments) {
-                await this.env.store.dispatch('fetchThreadAttachments', threadLocalId);
-            }
-            return threadLocalId;
-        };
-        this.createChatterTopbar = async (threadLocalId, otherProps) => {
+        this.createChatterTopbar = async (chatterLocalId, otherProps) => {
             ChatterTopBar.env = this.env;
-            this.chatterTopbar = new ChatterTopBar(null, Object.assign({ threadLocalId }, otherProps));
+            const defaultProps = {
+                isComposerLog: false,
+                isComposerVisible: false
+            };
+            this.chatterTopbar = new ChatterTopBar(null, Object.assign({ chatterLocalId }, defaultProps, otherProps));
```

Comment:
multi-line, since too long.
(I know, that's ugly... would be so nice using spread operator on object... Maybe we are now allowed to do so, since Edge Chromium released yesterday? Would talk with JS team and AL about this)